.husky/pre-commit

```diff
@@ -1 +1 @@
-npx lint-staged
+npx nano-staged
```

Comment:
Are we sure this change is the direction we want to go?  Just looking at adoption and project stats: 50k weekly downloads vs 14 million, and it's not reached 1.0 yet?  I like the work the e18e folks are doing in evangelizing package modernization, but they can be a bit too aggressive sometimes imho.  I'm not sure that a lint plugin enforcing their recommendations is the way to go.

Reply:
Yes please